OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -265,20 +264,17 @@ void LiveFunctionsDataView::OnContextMenu(const std::string& action, int menu_in
         live_functions_->AddIterator(selected_function);
       }
     }
-  } else if (action == kMenuActionFrameTrack) {
+  } else if (action == kMenuActionEnableFrameTrack) {
     for (int i : item_indices) {
       FunctionInfo* function = GetSelectedFunction(i);
-      const FunctionStats& stats = capture_data.GetFunctionStatsOrDefault(*function);
-      if (stats.count() > 1 && !GOrbitApp->HasFrameTrack(*function)) {
-        live_functions_->AddFrameTrack(*function);
-      }
+      GOrbitApp->EnableFrameTrack(*function);
```

Comment:
What happens to the next capture, if we unselect a function for the next capture, but add the frame track afterwards. The frame track will not appear in the next capture, right?

Reply:
This is actually a mistake. When the function is not hooked, it should get hooked here.

Reply:
Fixed.